tests/common_tools.py

```diff
@@ -124,10 +124,10 @@ def sequence_equals(sequence1, sequence2):
     return True
 
 
-def make_user():
+def make_user(**kwargs):
     user = auth_factories.UserFactory(
         email='joedoe@example.com', is_active=True,
-        is_staff=True  # TEMPORARY FOR PDF DEBUG TESTS
+        **kwargs
```

Comment:
in the csv export for the accountant, we check if the user is a staff member. I think we will fix some tests ;-)